torchgeo/samplers/constants.py

```diff
@@ -0,0 +1,6 @@
+from torch.utils.data import Sampler
+
+Sampler.__module__ = "torch.utils.data"
+
+SIZE_IN_PIXELS = 0
+SIZE_IN_CRS_UNITS = 1
```

Comment:
Python actually has an `enum` module for this, not sure if it would be beneficial or not